arlo-client/src/components/CreateAudit.tsx

```diff
@@ -44,17 +47,48 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
   return (
     <Wrapper>
       <img height="50px" src="/arlo.png" alt="Arlo, by VotingWorks" />
-      <Button
-        type="button"
-        intent="primary"
-        fill
-        large
-        onClick={onClick}
-        loading={loading}
-        disabled={loading}
-      >
-        Create a New Audit
-      </Button>
+      {isAuthenticated ? (
+        meta!.type === 'audit_admin' ? (
+          <Button
+            type="button"
+            intent="primary"
+            fill
+            large
+            onClick={onClick}
+            loading={loading}
+            disabled={loading}
+          >
+            Create a New Audit
```

Comment:
as per a discussion with @mcchilders, let's let a non-authenticated user create a new audit, too. So the create audit should be functional in both cases, not-logged-in and logged-in.

Reply:
@benadida Gotcha! Can do.

Reply:
@benadida Should audits created by a non-authenticated user be publicly accessible, like the old flow?